steps/linux_steps.py

```diff
@@ -0,0 +1,381 @@
+#!/usr/bin/env python3
+# -*- coding: ascii -*-
+import logging
+
+from utils.ssh_client import SSHClient
+
+logger = logging.getLogger(__name__)
+
+
+class LinuxSteps(SSHClient):
+
+    def __init__(self,
+                 host=None,
+                 username=None,
+                 password=None,
+                 key=None,
+                 passphrase=None,
+                 add_system_known_hosts=False,
+                 **kwargs):
+
+        super().__init__(host, username, password, key, passphrase, **kwargs)
+        if add_system_known_hosts:
+            self.add_system_known_hosts()
+
+    @staticmethod
+    def signature_creator(command, options):
+        signature = [command]
+        for option in options:
+            if option is None or not options[option]:
+                continue
+            else:
+                signature.append(option)
+        return ' '.join(signature)
+
+    def step(self, signature, get_signature=False, **kwargs):
+        if get_signature:
```

Comment:
```get_signature``` - remove it.
You have to add ```cd=None``` parameter. And if it is not None you will change directory before command. But it's dubious necessity.

Reply:
Done